tests/integration_tests/clouds.py

```diff
@@ -143,12 +143,30 @@ def _get_cloud_instance(self):
         )
 
 
+def _mount_source(instance):
```

Comment:
When I first read this, I thought it was potentially a candidate to move onto `IntegrationLxdInstance`

```suggestion
def _mount_source(pycloudlib_instance):
```

or, perhaps even better:

```suggestion
def _mount_source(instance: LXDInstance) -> None:
```

(plus an import).